articles/event-grid/system-topics.md

```diff
@@ -52,8 +52,12 @@ You can create a system topic in two ways:
 
 When you use [CLI](create-view-manage-system-topics-cli.md), [REST](/rest/api/eventgrid/version2020-06-01/eventsubscriptions/createorupdate), or [Azure Resource Manager template](create-view-manage-system-topics-arm.md), you can choose either of the above methods. We recommend that you create a system topic first and then create a subscription on the topic, as this is the latest way of creating system topics.
 
+### Failure to create system topics
 The system topic creation fails if you have set up Azure policies in such a way that the Event Grid service can't create it. For example, you may have a policy that allows creation of only certain types of resources (for example: Azure Storage, Azure Event Hubs, etc.) in the subscription. 
 
+In such case, Event flow functionality is preserved, however Metrics and Diagnostic functionalities of System Topics will be unavailable.
+If you require this functionality, allow System Topics resource type and create the missing System Topic as [described above](#lifecycle-of-system-topics).
```

Comment:
If you require this functionality, allow creation of resources of the system topic type, and create the missing system topic as described in the [Life cycle of system topics](#lifecycle-of-system-topics) section.